### PR TITLE
345 split train test data in correct way

### LINKED ISSUE #345: Split train test data in correct way
**Reason (Why?)**
The data set is split by using the first 80% for training and the last 20% for test. This can lead to errors.

**Solution (What?)**
Use another method to split the data so that the existing labels are present both in the test and in the training.

**Acceptance criteria**
Data sets where a label has fewer entries should have all existing labels in the test and in training

**Idea**
Use scikit-learn

### PR DESCRIPTION
**Pull request by piafriehe**
Splits the training and test data so that all classes are represented in the training and test data and the distribution of all classes in the test set is the same as in the training set.

Undo the fix in the AdapterUtils because it is no longer need

Closes #345

### CODE REVIEW

**Review by AlexanderZender — APPROVED** [review]
awesome!
